docs/Modders/Images-and-Audio.md

```diff
@@ -41,6 +41,14 @@ These work best if they are square, between 100x100 and 256x256 pixels, and incl
 
 For example, [here](https://github.com/yairm210/Unciv-leader-portrait-mod-example) is mod showing how to add leader portraits, which can complement the base game.
 
+### Adding Unit and Building Portraits
+
+The base game comes without Unit/Building Portraits (display flags instead by default), but is able to display them in City Screen, Overview Screen, Tech Tree, Civilopedia, etc. A mod can supply these, by adding their images to `/Images/UnitPortraits/` and `/Images/BuildingPortraits`. The file name must correspond exactly with the unit/building name as defined in Units.json and Buildings.json, or they will be ignored.
```

Comment:
```suggestion
The base game comes without Unit/Building Portraits (display flags instead by default), but is able to display them in City Screen, Overview Screen, Tech Tree, Civilopedia, etc. A mod can supply these, by adding their images to `/Images/UnitPortraits/` and `/Images/BuildingPortraits/`. The file name must correspond exactly with the unit/building name as defined in Units.json and Buildings.json, or they will be ignored.
```

Reply:
Actually this doesn't describe what 'flag' vs' portrait' is.

Something like:

The base game uses flat icons, colored to fit the civilization's flag colors, to denote units both on the map and in other UI elements.
A mod can supply "Portraits" - static images that will remain uncolored -  by adding their images to `/Images/UnitPortraits/`, which will be used in all UI elements except for the world map. The file name must correspond exactly with the unit/building name as defined in Units.json and Buildings.json, or they will be ignored.